src/api/index.ts

```diff
@@ -5,14 +5,15 @@ import mappingRouter from './mapping'
 import sshKeyRouter from './sshKeys'
 import accessTokensRouter from './accessToken'
 import { getAvailableDomains } from '../lib/data'
+import { validUser, validAdmin } from '../helpers/auth'
 
 const apiRouter = express.Router()
 
-apiRouter.use('/admin', adminRouter.app)
-apiRouter.use('/logs', logsRouter)
-apiRouter.use('/mappings', mappingRouter)
-apiRouter.use('/sshKeys', sshKeyRouter)
-apiRouter.use('/accessTokens', accessTokensRouter)
+apiRouter.use('/admin', validAdmin, adminRouter.app)
```

Comment:
`validAdmin` should be used in the `adminRouter` as a middleware instead of here.